resources/definitions/makeR_prusa_tairona_i3.def.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "makeR_prusa_tairona_i3",
+    "version": 3,
```

Comment:
This needs to be a 2. The version number is what version the way we handle definition files is. So currently the entire file is ignored by Cura.